todo/src/components/AddTodo.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from "react"
+import { todoType } from "../App"
+
+const AddTodo = ({todoArr,addTask}:{todoArr: todoType[],addTask: (taskName: string) => void}) => {
+    const[str,setStr] = useState<string>('')
```

Comment:
what is str? Please use proper naming convention for state also add comment for states